From 849b001d5bd4b979c9335fe8e53e58bfe40dbcb3 Mon Sep 17 00:00:00 2001
From: Yinhang Liu <yinhang.liu@intel.com>
Date: Tue, 18 Apr 2023 17:08:54 +0800
Subject: [PATCH 110/232] msdk: Remove unreachable statement

The execution cannot reach the statement: "gst_caps_unref(dma_caps);",
so remove it.

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/4449>
---
 sys/msdk/gstmsdkcaps.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/sys/msdk/gstmsdkcaps.c b/sys/msdk/gstmsdkcaps.c
index 2a9632607..5d9988aa5 100644
--- a/sys/msdk/gstmsdkcaps.c
+++ b/sys/msdk/gstmsdkcaps.c
@@ -1728,8 +1728,6 @@ failed:
     gst_caps_unref (in_caps);
   if (out_caps)
     gst_caps_unref (out_caps);
-  if (dma_caps)
-    gst_caps_unref (dma_caps);
 
   return FALSE;
 }
-- 
2.40.1

